LibreNMS/ObjectCache.php

```diff
@@ -113,34 +114,30 @@ public function offsetGet($obj)
     /**
      * Overrides internal Cache-Object
      *
-     * @param  string  $obj  Name of Data-Object
+     * @param  mixed  $obj  Name of Data-Object
      * @param  mixed  $value  Value
-     * @return bool
+     * @return void
      */
-    public function offsetSet($obj, $value)
+    public function offsetSet($obj, $value): void
     {
         if (! isset($this->data[$obj])) {
             $this->data[$obj] = [];
         }
 
         $this->data[$obj]['value'] = $value;
-
-        return $this->data[$obj]['value'];
```

Comment:
It's tricky to find if this will break something

Reply:
It should not